applications/serial_lte_modem/doc/nRF91_as_Zephyr_modem.rst

```diff
@@ -0,0 +1,130 @@
+.. _slm_as_zephyr_modem:
+
+nRF91 Series as a Zephyr-compatible modem
+#########################################
+
+.. contents::
+   :local:
+   :depth: 2
+
+Overview
+********
+
+Starting with |NCS| v2.6.0, the Serial LTE Modem (SLM) application can be used to turn an nRF91 Series SiP into a standalone modem that can be used through Zephyr's cellular modem driver.
+This means that the controlling chip can run a Zephyr application that seamlessly uses Zephyr's IP stack instead of offloaded sockets via AT commands.
+
+This is made possible by SLM's support of CMUX and PPP and Zephyr's cellular modem driver.
+
+The :ref:`nRF52840 SoC present in the nRF9160 DK <zephyr:nrf9160dk_nrf52840>` can even be made to be the controlling chip with the appropriate configuration (described below).
+This is only possible on the nRF9160 DK, not on the other nRF91 Series DKs.
+
+.. note::
+
+   As of |NCS| v2.6, Zephyr's GNSS driver does not support the nRF91 Series running SLM.
```

Comment:
```suggestion
   As of |NCS| v2.6.0, Zephyr's GNSS driver does not support the nRF91 Series running SLM.
```

Reply:
I put 2.6 and not 2.6.0 to clearly indicate that there is no chance any such change would come in a minor release (e.g. 2.6.1).